src/network/MissionControlTasks.cpp

```diff
@@ -185,4 +242,23 @@ void ArmIKTask::updateArmIK() {
 	}
 }
 
+static DataPoint<control::swerve_rots_t> getAllSteerRotations() {
+	try {
+		return DataPoint<control::swerve_rots_t>(
+			{robot::getMotorPos(motorid_t::frontLeftWheel).getData(),
+			 robot::getMotorPos(motorid_t::frontRightWheel).getData(),
+			 robot::getMotorPos(motorid_t::rearLeftWheel).getData(),
+			 robot::getMotorPos(motorid_t::rearRightWheel).getData()});
+	} catch (const std::runtime_error& e) {
```

Comment:
Should this be  `bad_datapoint_access` instead of `std::runtime_error` once that PR is merged?

Reply:
Yessir